quartz/plugins/emitters/aliases.ts

```diff
@@ -9,9 +9,30 @@ export const AliasRedirects: QuartzEmitterPlugin = () => ({
   getQuartzComponents() {
     return []
   },
-  async getDependencyGraph(_ctx, _content, _resources) {
-    // TODO implement
-    return new DepGraph<FilePath>()
+  async getDependencyGraph(ctx, content, _resources) {
+    const graph = new DepGraph<FilePath>()
+
+    const { argv } = ctx
+    for (const [_tree, file] of content) {
+      const dir = path.posix.relative(argv.directory, path.dirname(file.data.filePath!))
+      const aliases = file.data.frontmatter?.aliases ?? []
+      const slugs = aliases.map((alias) => path.posix.join(dir, alias) as FullSlug)
+      const permalink = file.data.frontmatter?.permalink
```

Comment:
is this a new feature? afaik we don't process permalink anywhere?

I might be wrong, please do correct me.

Reply:
We do, code is copied from the emit method.